modules/mobile/app/controllers/mobile/v0/military_information_controller.rb

```diff
@@ -29,10 +26,6 @@ def military_info_adapter
       def service
         VAProfile::MilitaryPersonnel::Service.new(user)
       end
-
-      def handle_errors!(response)
-        raise_error! unless response.is_a?(Array)
```

Comment:
Removing this because it was copy/pasted from a controller elsewhere in the vets-api but the `raise_error!` method wasn't included. The fact that we're not seeing issues related to this tells me it must not be happening. So I think we can just let it fail normally and we'll see 500s.